setup.py

```diff
@@ -1,4 +1,5 @@
-# Copyright (C) 2015 UCSC Computational Genomics Lab
+#
+# Copyright 2015–2016, Regents of the University of California
```

Comment:
I generally prefer the `Regents of the University of California` copyright—esp. if you have contributors from multiple UCs—but I'm not picky.